dynamic-plugins/imports/package.json

```
@@ -27,8 +27,8 @@
     "@janus-idp/backstage-plugin-quay": "1.11.3",
     "@janus-idp/backstage-plugin-rbac": "1.29.4",
     "@janus-idp/backstage-plugin-tekton": "3.12.3",
-    "@janus-idp/backstage-plugin-bulk-import": "1.4.6",
-    "@janus-idp/backstage-plugin-bulk-import-backend": "1.4.1",
+    "@janus-idp/backstage-plugin-bulk-import": "1.4.7",
+    "@janus-idp/backstage-plugin-bulk-import-backend": "1.4.3",
```

```suggestion
    "@janus-idp/backstage-plugin-bulk-import-backend": "1.5.0",
```

New version of the bulk import backend plugin available; it adds server-side search https://issues.redhat.com/browse/RHIDP-3884 (https://github.com/janus-idp/backstage-plugins/pull/2140)